CHANGELOG.md

```diff
@@ -43,6 +43,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Adds helper `associateExecutionWithGranule` to `@cumulus/api-client/granules`
 - **CUMULUS-2583**
   - Adds `queued` as option for granule's `status` field
+- **CUMULUS-2630**
```

Comment:
This changelog note shouldn't be nested under 9.6.0

Reply:
833f3cd6b7